packages/melos/lib/src/common/utils.dart

```diff
@@ -255,6 +255,8 @@ Future<int> startProcess(
     return _arg;
   }).where((element) => element != null);
 
+  if (logger.isVerbose) environment.addAll({'MELOS_VERBOSE': 'ture'});
```

Comment:
```suggestion
  if (logger.isVerbose) environment.addAll({'MELOS_VERBOSE': 'true'});
```

const for the env key as mentioned in earlier comment - thanks